tests/Service/TagsServiceTest.php

```diff
@@ -0,0 +1,71 @@
+<?php
+
```

Comment:
Is there a reason why there are two empty lines here?

Reply:
adjusted